styles/bootswatch.less

```diff
@@ -1,237 +0,0 @@
-// Cosmo 3.3.2
```

Comment:
I guess it should be there since it is referenced by main.less... or is it pulled down automatically?